<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifferenceMap</class>
 <widget class="QWidget" name="DifferenceMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1">
   <item row="0" column="1">
    <widget class="QLineEdit" name="refImgText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="loadRefImgButton">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="refImgName">
     <property name="text">
      <string>Reference Image: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="thresholdSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="thresholdLabel">
     <property name="text">
      <string>Threshold: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="thresholdText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
